src/main/java/com/example/board/domain/member/entity/Member.java

```diff
@@ -0,0 +1,72 @@
+package com.example.board.domain.member.entity;
+
+import com.example.board.domain.common.entity.BaseTimeEntity;
+import com.example.board.domain.post.entity.Post;
+import java.util.ArrayList;
+import java.util.List;
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.FetchType;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+import javax.persistence.OneToMany;
+import javax.persistence.Table;
+import javax.persistence.UniqueConstraint;
+
+@Entity
+@Table(uniqueConstraints = {
+    @UniqueConstraint(name = "NAME_UNIQUE", columnNames = {"name"})})
+public class Member extends BaseTimeEntity {
+
+  @Id
+  @GeneratedValue(strategy = GenerationType.SEQUENCE)
+  @Column(name = "member_id")
+  private Long id;
+
+  @Column(name = "name")
+  private String name;
+
+  @Column(name = "age")
+  private int age;
+
+  @Column(name = "hobby")
+  private String hobby;
+
+  @OneToMany(mappedBy = "member", fetch = FetchType.LAZY)
+  private List<Post> posts = new ArrayList<>();
+
+  protected Member(){}
+
+  public Member(String name, int age, String hobby) {
+    this.name = name;
+    this.age = age;
+    this.hobby = hobby;
+  }
+
+  public void update(String newName, int newAge, String newHobby){
+    name = newName;
+    age = newAge;
+    hobby = newHobby;
+  }
+
+  public Long getId(){
+    return id;
+  }
+
+  public String getName(){
+    return name;
+  }
+
+  public int getAge(){
+    return age;
+  }
+
+  public String getHobby(){
+    return hobby;
+  }
+
+  public List<Post> getPosts(){
+    return posts;
+  }
```

Comment:
```suggestion
  public List<Post> getPosts(){
    return new ArrayList<>(posts);
  }
```

처럼 방어적 복사를 해주세요. 불변 리스트를 반환하길 원한다면 그것도 좋겠네요

Reply:
방어적 복사 좋네요! 적용해보겠습니다

Reply:
@seung-hun-h 
좀 더 생각을 해봤는데요, 방어적 복사는 API를 사용하는 쪽에서 참조값을 이용해 변형을 하는 것을 막으려고 사용하는 것 같습니다. 
그런데 방어적 복사를 적용하면 API 내부를 직접 까보기 전까지는 이 객체를 가지고 아무런 변형을 가할 수 없다는 것을 API 사용자가 알 방법이 없습니다.

방어적 복사는 좋은 것 같지만, 가변 객체를 직접 참조하지 않아야 한다는 것을 API 사용자에게 알려줄 방법이 없을까요?